packages/spark-extras/components/card/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@sparkdesignsystem/spark-card",
```

Comment:
@RCopeland 
I looked at dictionary and highlight board and they use ""name": "@sparkdesignsystem/spark-description-table"," To be consistent is why i went with the same naming.

Reply:
the name comes from spark-core and spark-extras having the name prefix, which I think was from before everything was under the @sparkdesignsystem org in npm. You can't actually change the name of a package. Have to create a new one and deprecate the old one because deleting is weird. So I would just keep the naming like that. little more verbose, but not a big issue.

Reply:
I remember you mentioning that. Security issue. Cool, I’ll leave as I had it.